bin/check-log.rb

```diff
@@ -85,13 +86,13 @@ class CheckLog < Sensu::Plugin::Check::CLI
          long: '--encoding ENCODING-PAGE'
 
   option :warn,
-         description: 'Warning level if pattern has a group',
+         description: 'Warning level if pattern is found >= N times',
          short: '-w N',
          long: '--warn N',
          proc: proc(&:to_i)
 
   option :crit,
-         description: 'Critical level if pattern has a group',
+         description: 'Critical level if pattern is found >= N times',
```

Comment:
If this is true then we should probably make it a default of `1` as no default will eval to `nil` when calling `.to_i` it becomes `0` so it should either be a default of 1 or remove the greater equality for a greater than.
```
$ irb
irb(main):001:0> nil.to_i
=> 0
```